docs/feature-flags/flag-variations.md

```diff
@@ -4,8 +4,30 @@ sidebar_position: 4
 
 # Flag variations
 
-All Eppo flags return strings. However, since strings are generic, they can be used to represent numbers (e.g `'5'`), booleans (e.g `'true'`), and even maps with arbitrary key-value pairs:
+Eppo flags support multiple types: boolean, string, numeric, and JSON.
 
-![Typed flags](/img/feature-flagging/typed-flags.png)
+## Boolean flags
 
-In the example above the flag has two variants, each with it’s own JSON value represented as a string. When delivering this flag, _Eppo SDKs will return strings_. It is up to your application to deserialize these strings into maps. The same is true for numbers and booleans. Deserialized typed values can than then be used in your application logic.
+Boolean flags can only ever have two variants, each corresponsing to a `true` or `false` value.We recommend setting boolean flags to `true` when the flag is on and `false` when the flag is off.
+
+## String flags
+
+String flags have a limit of 32KB. Strings are UTF-8 encoded, so certain characters, like emojis and non-ASCII characters, take more than one byte each.
```

Comment:
Our limit is 65 KB (postgres `text` field`. We can increase this to practically any limit in the future if needed.